src/components/split-button/split-button-test.stories.tsx

```diff
@@ -53,44 +54,39 @@ export default {
   },
 };
 
-/* eslint-disable react/prop-types */
+type SplitButtonStoryArgs = {
```

Comment:
Since these prop types are the same as in `SplitButton` (other than `textContent` which doesn't seem to be used - see below), I think it's better to write this as:

`type SplitButtonStoryArgs = Pick<SplitButtonProps, "buttonType" | "text" | "subtext">;`

to be explicit that we're just picking particular props from the `SplitButtonProps`. This also saves us having to update this type separately if we ever decide to change the type of one of these props (which is very unlikely, I admit!)